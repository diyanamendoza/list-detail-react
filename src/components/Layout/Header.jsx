import { Link } from "react-router-dom"


export default function Header() {
    return (
        <header>
            <Link to='/'>
            <h1>Avatar: The Last Airbender</h1>
            <span>--------------</span>
            <h2>Character List</h2>
            </Link>
        </header>
    )
}
