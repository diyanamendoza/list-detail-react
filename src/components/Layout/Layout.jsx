import React from 'react'
import Header from './Header'
import styles from '../../App.css'

export default function Layout({children}) {
    return (
        <div className={styles.layout}>
            <Header />
            {children}
        </div>
    )
}
