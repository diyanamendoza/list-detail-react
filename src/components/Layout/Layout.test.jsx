import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom';
import Layout from './Layout'

it('should render the layout', () => {
    const { container } = render(
        <MemoryRouter>
            <Layout />
        </MemoryRouter>
        )
    expect(container).toMatchSnapshot()
})