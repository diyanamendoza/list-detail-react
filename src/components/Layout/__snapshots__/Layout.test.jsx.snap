// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the layout 1`] = `
<div>
  <div
    class="layout"
  >
    <header>
      <a
        href="/"
      >
        <h1>
          Avatar: The Last Airbender
        </h1>
        <span>
          --------------
        </span>
        <h2>
          Character List
        </h2>
      </a>
    </header>
  </div>
</div>
`;
